src/main/java/org/jenkinsci/maven/plugins/hpi/RunMojo.java

```diff
@@ -605,93 +600,30 @@ private String loadVersion(InputStream is) throws IOException {
     }
 
     @Override
-    public void configureScanner() throws MojoExecutionException {
+    public void startScanner() throws Exception {
```

Comment:
Adapting the hack to changes in the superclass's calling conventions. See the comment for explanation.